src/commands/transaction/create/transfer.js

```diff
@@ -19,10 +19,22 @@ import BaseCommand from '../../../base';
 import commonFlags from '../../../utils/flags';
 import getInputsFromSources from '../../../utils/input';
 
-const processInputs = (amount, address) => ({ passphrase, secondPassphrase }) =>
+const dataFlag = {
+	char: 'd',
+	description: `Message of UTF-8 string to attach, which has a maximum length of 64 bytes.
```

Comment:
I feel like this sentence doesn't make much sense...
@mitsujutsu can help with the better english i think

Reply:
Oh I thought you wanted to use this sentence only